docs/source/upcoming_release_notes/1153-add_spectrometer_solid_attenuator.rst

```diff
@@ -0,0 +1,30 @@
+1153 add spectrometer solid attenuator
+#################
+
+API Changes
+-----------
+Adding new TMOSpectrometerSOLIDATTStates class and thorlab_len_X axis in TMOSpectrometer
```

Comment:
Hi Tong, I'm going to hit approve, but you should know for next time that this isn't the right categorization/formatting.

- You need to keep the original template formatting where each line starts with a `- ` so that the rst renders as a bulleted list.
- Check out the category explanations at https://github.com/pcdshub/pcdsdevices/blob/master/docs/source/upcoming_release_notes/template-full.rst
  - So, adding stuff is never an API change. The only API change in this PR is probably the removal of the lens motors
  - All the other changes should be "Device Updates"

You can fix these if you want, or just keep in mind for next time and l'll fix them when I assemble the full release note later.

Reply:
I can change again and push again, Zach.